core/src/main/java/dk/alexandra/fresco/lib/compare/MiscBigIntegerGenerators.java

```diff
@@ -90,9 +90,9 @@ public MiscBigIntegerGenerators(BigInteger modulus) {
 
     for (int i = 1; i <= l; i++) {
       int k = i;
-      if (i >= m) {
```

Comment:
Again, if this is the case, just remove it. Can keep the actual comment though as that might be useful.